ceph_deploy/osd.py

```diff
@@ -674,6 +675,7 @@ def make(parser):
     parser.description = sub_command_help
 
     osd_parser = parser.add_subparsers(dest='subcommand')
+    osd_parser.required = True
```

Comment:
Python 3.4 does not give an error when an incomplete command is passed, e.g. `ceph-deploy mon`, making it crash with an exception later down the line.

More info: http://bugs.python.org/issue9253#msg186387

Minimal example to compare behavior of Pythons:
```python
import argparse
parser = argparse.ArgumentParser()
parser.add_subparsers()
parser.parse_args(args=[])
```